src/test/java/com/google/api/codegen/testdata/java_test_library.baseline

```diff
@@ -171,11 +171,11 @@ public class LibraryServiceTest {
   @Test
   @SuppressWarnings("all")
   public void getShelfTest() {
-    String formattedName2 = LibraryServiceClient.formatShelfName("[SHELF_ID]");
+    String name2 = "";
```

Comment:
This does not look correct. The assigned value should be a randomized string (not empty).